Recrutify/Recrutify.Services/Services/SendEmailService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MailKit.Security;
+using Microsoft.Extensions.Options;
+using MimeKit;
+using Recrutify.DataAccess.Configuration;
+using Recrutify.DataAccess.Models;
+using Recrutify.Services.Services.Abstract;
+
+namespace Recrutify.Services.Services
+{
+    public class SendEmailService : ISendEmailService
+    {
+        private readonly MailSettings _mailSettings;
+        private readonly IFormEmailService _formEmailService;
+
+        public SendEmailService(IOptions<MailSettings> mailSettings, IFormEmailService formEmailService)
+        {
+            _mailSettings = mailSettings.Value;
+            _formEmailService = formEmailService;
+        }
+
+        public void SendEmail(EmailRequest emailRequest)
+        {
+            var email = new MimeMessage();
+
+            email.Sender = MailboxAddress.Parse(_mailSettings.Mail);
```

Comment:
why do you parse same data twice? put it in separate var sender = MailboxAddress.Parse(_mailSettings.Mail) and use in lines 25-26